tests/TestCase/View/ViewVarsTraitTest.php

```diff
@@ -186,12 +186,10 @@ public function testUptoDateViewVars()
      */
     public function testViewOptionsGetsToBuilder()
     {
-        $this->deprecated(function () {
-            $this->subject->passedArgs = 'test';
-            $this->subject->createView();
-            $result = $this->subject->viewBuilder()->getOptions();
-            $this->assertEmpty($result);
-        });
+        $this->subject->passedArgs = 'test';
```

Comment:
Don't see the point of setting an unused property. Either it should be replaced by equivalent new method or the whole test method should be removed.

Reply:
>test that removed deprecated passedArgs is no longer passed to the view

Reply:
>should be removed.

:+1: